isup/isup-impl/src/main/java/org/mobicents/protocols/ss7/isup/util/BcdHelper.java

```diff
@@ -0,0 +1,119 @@
+/**
+ *  Copyright 2016 Grzegorz Figiel
+ *
```

Comment:
Yes I did modified the original. I described the modification and link to original file in Class comments.